resume_agent/backend/document_processor.py

```
@@ -0,0 +1,120 @@
+        Args:
+            file_paths: List of file paths to parse
+
+        Returns:
+            List of dictionaries containing parsed documents
+        """
+        parsed_docs = []
+        for file_path in file_paths:
+            try:
+                doc = self.parse_document(file_path)
+                parsed_docs.append(doc)
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)